<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>جدید</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ثبت</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>خروج</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>وضعیت تاهل :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>           درجه</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>     شماره ملی</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>  شماره پرسنلی</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>    مدرک تحصیلی</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>          نام پدر</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>             نام</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>            نشان</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>     شغل عملی</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>0</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>مشخصات فردی</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره شناسنامه</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>611</x>
       <y>120</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>مجرد</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>            یگان</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>       گروه خون</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>    شغل سازمانی</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>541</x>
       <y>120</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>متاهل</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>781</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>            نشان</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>       تاریخ تولد</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>              نام</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره شناسنامه</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>     شماره ملی</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>مشخصات همسر :</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>فرزند دارد</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>فرزند ندارد</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>مشخصات فرزندان :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>تعداد عائله تحت تکفل :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_18">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>مستاجر</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>شخصی</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>سازمانی</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>0</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>نوع منزل :</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>دارم</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>656</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>بیمار صعب العلاج :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_9">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ندارم</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>231</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>فرزند</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>مربوط به :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>همسر</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>وارد کردن طبقه پزشکی</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>781</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_28">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>70</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره جساب حکمت همسر</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_26">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>70</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره حساب حکمت</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_23">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>            شماره تماس</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_30">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره کارت سپه</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_27">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره کارت حکمت</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_31">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>100</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره کارت حکمت همسر</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_25">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>      شماره تماس منزل</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_29">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>100</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>شماره حساب سپه</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_24">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>491</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>   آدرس شهرستان</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_22">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>491</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>   آدرس قزوین</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
